app/helpers/application_helper.rb

```diff
@@ -270,12 +270,6 @@ def display_estimated_user_count?
   def admin_config_label(method, content = nil, model: Settings::General)
     content ||= tag.span(method.to_s.humanize)
 
-    if method.to_sym.in?(Settings::Mandatory.keys)
```

Comment:
We want to remove this [Settings::Mandatory Module](https://github.com/forem/forem/blob/juliannatetreault/remove-get-started-section-from-config/app/models/settings/mandatory.rb) as well